package.json

```diff
@@ -28,7 +28,7 @@
     "build:ci": "tsc -p tsconfig.json",
     "build:swagger": "npm run build && npm run setup:swagger",
     "build": "npm run build:ci && npm run copy",
-    "build:clean": "npm run clean && npm run build",
+    "clean:build": "npm run clean && npm run build",
```

Comment:
keep both for backward compatibility

Reply:
don't see any reference of `build:clean` in docker compose, so nothing should break. But if someone using it in locally, will affect them

Reply:
Addressed @koladilip